### DISCUSSION TITLE
output not come from naabu

### DISCUSSION THREAD

**Discussion by d4715** [projectdiscovery/naabu#190] [Q&A] [answered]
Hello there,
Today i am first time install naabu for port scan.
but after install successfully naabu i did not got perfect port scan result.
First i share my installation steps with you.
Before installing naabu, I installed libpcap library with apt install -y libpcap-dev on Linux

1) First Method 
 1.1)GO111MODULE=on go get -v github.com/projectdiscovery/naabu/cmd/naabu

2) Second Method
 2.1)Download latest file :> wget https://github.com/projectdiscovery/naabu/releases/download/v2.0.4/naabu_2.0.4_linux_amd64.zip
 2.2)extract naabu_2.0.4_linux_amd64.zip
 2.3)cp naabu /usr/local/bin
 2.4)cp naabu /root/go/bin 

3) Third Method 
 3.1)git clone https://github.com/projectdiscovery/naabu.git
 3.2)cd naabu/v2/cmd/naabu
 3.3)go build
 3.4)cp naabu /usr/local/bin/

I used these 3 methods  one by one for install naabu and i installed naabu successfully but did not get any successful result
here i want to share my commands for get port scan results  via naabu 

1)naabu -v -iL 200.txt | tee 200portscan.txt
result in image 
![1](https://user-images.githubusercontent.com/11803332/123032130-14917700-d403-11eb-98c6-00e9fc1fb53e.JPG)

2)naabu -v -exclude-ports 80,443 -iL 200.txt | tee 200portscan.txt 
result in image 
![2](https://user-images.githubusercontent.com/11803332/123032549-b9ac4f80-d403-11eb-8580-9b9c37f5a2d8.JPG)

i can not get any result so please help me to solve this issue

I am using latest and updated kali linux in vmware version 14.1.3 build-9474260 
kali linux details:
PRETTY_NAME="Kali GNU/Linux Rolling"
NAME="Kali GNU/Linux"
ID=kali
VERSION="2021.2"
VERSION_ID="2021.2"
VERSION_CODENAME="kali-rolling"
ID_LIKE=debian
ANSI_COLOR="1;31"
HOME_URL="https://www.kali.org/"
SUPPORT_URL="https://forums.kali.org/"
BUG_REPORT_URL="https://bugs.kali.org/" 

Thanks & Regards, 
D4715

**Comment by ehsandeep** [comment]
@d4715 to debug this further, we might need input details, can you attach them here if it's not private, otherwise DM us in the [discord channel](https://discord.gg/projectdiscovery) to "pdteam"

**Comment by d4715** [comment]
Hello There,
Thank you for your response .
you need some details means you need domains name?
if yes so i tried lots of domains.
domains list (all domains taken from this github repo :> https://github.com/Sajibekanti/Bug_Bounty_List)
https://konnect.nl
https://www.infomedics.nl
https://greenhome.nl
https://www.cumlaudevo.nl

1)naabu -v -iL 200.txt | tee 200portscan.txt
result in image
![1](https://user-images.githubusercontent.com/11803332/123043542-0dc02f80-d416-11eb-9a88-82f00cfc6516.JPG)

2)naabu -v -exclude-ports 80,443 -iL 200.txt | tee 200portscan.txt
result in image
![2](https://user-images.githubusercontent.com/11803332/123043563-1add1e80-d416-11eb-93ae-f31a9d79144c.JPG)

Thanks and Regards,
D4715

**Comment by d4715** [comment]
Hello There,
can i get any update?
Thanks and Regards,
D4715

**Comment by Mzack9999** [comment]
It seems like the error is due to the fact that in your file all fqdn names are prefixed with `http://` or `https://`. Could you try to create another file with the prefixes removed and run the tool against it?

**Reply by d4715** [reply]
Hello There, 
OK i got your point . if any doubts occurred i will ping you back .
Thanks & Regards,
D4715

**Comment by ehsandeep** [comment, marked as answer]
@d4715 I can confirm it's an issue with the input file format, the expected format is 

```
konnect.nl
www.infomedics.nl
greenhome.nl
```

instead of 

```
https://konnect.nl
https://www.infomedics.nl
https://greenhome.nl
```

With correct input, here is an example output.

```
cat a.txt | naabu -p 80 -silent
konnect.nl:80
greenhome.nl:80
www.infomedics.nl:80
```

**Reply by d4715** [reply]
Hello There, 
Thanks for solutions . if any doubts occurred i will ping you back .
Thanks & Regards,
D4715
